com.unity.uiwidgets/Runtime/cupertino/date_picker.cs

```diff
@@ -973,10 +1042,16 @@ class _CupertinoDatePickerDateState : State<CupertinoDatePicker> {
                     magnification: CupertinoDatePickerUtils._kMagnification,
                     backgroundColor: widget.backgroundColor,
                     squeeze: CupertinoDatePickerUtils._kSqueeze,
-                    onSelectedItemChanged: (int index) => {
+                    onSelectedItemChanged: index => {
+                        if (index < 0) {
```

Comment:
I don't see similar codes in flutter.

I believe the root cause is not here. We should investigate why index can be smaller than 0 ?